replica/database.cc

```diff
@@ -1031,9 +1031,9 @@ future<> database::add_column_family(keyspace& ks, schema_ptr schema, column_fam
         throw std::invalid_argument("Column family " + schema->cf_name() + " exists");
     }
     ks.add_or_update_column_family(schema);
-    cf->start();
     schema->registry_entry()->set_table(cf->weak_from_this());
     co_await _tables_metadata.add_table(schema);
+    cf->start();
```

Comment:
I'm not sure it's correct to add the table with starting it.
Why not add a deferred stop and cancel it before returning on success?